pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -24,21 +24,19 @@ def __init__(self, pokemon, bot):
         self.spawn_point_guid = ''
         self.response_key = ''
         self.response_status_key = ''
-        self.transfer_worker = None
 
     def work(self):
         encounter_id = self.pokemon['encounter_id']
         response_dict = self.create_encounter_api_call()
-        self.transfer_worker = PokemonTransferWorker(self)
 
         if response_dict and 'responses' in response_dict:
             if self.response_key in response_dict['responses']:
                 if self.response_status_key in response_dict['responses'][self.response_key]:
                     if response_dict['responses'][self.response_key][self.response_status_key] is 7:
                         if self.config.release_pokemon:
                             logger.log('Pokemon Bag is full!', 'red')
-                            self.transfer_worker.work()
-
+                            worker = PokemonTransferWorker(self)
+                            worker.work()
```

Comment:
@Nihisil a worker should never be called inside another worker like this. The `tick` function of the bot main class is already executing the transfer worker before pokemon catch worker. Please, remove this.